packages/rancher-alerting-drivers/charts/Chart.yaml

```diff
@@ -0,0 +1,44 @@
+name: rancher-alerting-drivers
+home: ""
+sources: []
+version: 1.0.1
+description: The manager for third-party webhook receivers used in the Alertmanger
+keywords:
+- monitoring
+- alertmanger
+- webhook
+maintainers:
+- name: Jiaqi Luo
+  email: ""
+  url: https://github.com/jiaqiluo
+icon: ""
+apiversion: v2
+condition: ""
+tags: ""
+appversion: 1.16.0
+deprecated: false
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: Alerting Drivers
+  catalog.cattle.io/experimental: "true"
+  catalog.cattle.io/os: linux
+  catalog.cattle.io/release-name: rancher-alerting-drivers
```

Comment:
Hmm, we have namespace in our required list, but I suppose it doesn't make sense because the user should be deploying this. It's really more of a partner chart that we own. 

That being said I think the UI specifically looks for a namespace to deploy to. Does this need to be in same namespace as the projects ? If so that might break this. If not we could do `cattle-alerting-system`

Reply:
Since we do not set the annotation for the namespace, so the UI will allow users to choose the namespace to install the app.